release/README.md

```diff
@@ -0,0 +1,22 @@
+# Release tools #
+
+This directory contains tools for Kata Containers releases.
+
+## tag_repos.sh ##
+
+The `tag_repos.sh` script is used to create tags for the Kata Containers
+repositories. This script make sure that all the repositories are in the
+same version (by checking the `VERSION` file).
+
+It will create an **annotated tag** for the new release version for the
+following repositories:
+
+- agent.
+- proxy.
+- runtime.
+- shim.
+- throttler.
+
+The script will also tag the tests and osbuilder repositories to make it
```

Comment:
Lines 20-22 suggested rewrite to keep active: The script also tags the tests and osbuilder repositories to make it clear which versions of these supporting repositories are used for the release.